### DISCUSSION TITLE
logstash-output-syslog

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#1666] [General]
Requesting this plugin be added to logstash container to facilitate forwarding logs to remote system, even after SO soup is run in airgap environment

**Comment by weslambert** [comment]
This probably needs to be discussed internally, as the syslog output plugin, to my knowledge, is not officially supported by Elastic.

**Comment by darth-drew** [comment]
That lines up with what I am seeing on the logstash-output-syslog.. looks like the build is failing? We are going to look at the kafka integration as an alternative.
